python/pyspark/rdd.py

```diff
@@ -159,7 +174,8 @@ class PyLocalIterable(object):
         """ Create a synchronous local iterable over a socket """
 
         def __init__(self, _sock_info, _serializer):
-            self._sockfile = _create_local_socket(_sock_info)
+            port, auth_secret, self.jserver_obj = _sock_info
```

Comment:
just a general comment, might not make the most sense to address in this particular PR -- I'd find it really helpful if the python code which is dealing w/ java objects would annotate (somehow) the java types.  Its hard for me to figure out if `jserver_obj` is a ServerSocket or a SocketAuthServer or Py4JJavaServer etc.

Reply:
sure, I can rename it to something more fitting and I agree it should be clear what the variable is by the name